app/assets/javascripts/discourse/app/lib/plugin-api.js

```diff
@@ -1850,6 +1850,7 @@ class PluginApi {
    * @param {string} arg.name - The name of the link. Needs to be dasherized and lowercase.
    * @param {string=} arg.route - The Ember route name to generate the href attribute for the link.
    * @param {string=} arg.href - The href attribute for the link.
+   * @param {string=} arg.icon - The FontAwesome icon to display for the link.
```

Comment:
What does the = means here?

Reply:
Oh ok I see it's optional, I think I have seen it done differently too. This syntax is google closure syntax, JSDoc syntax is `{string} [defaultvalue]` and I guess google closure should be `{string=} defaultvalue`.

https://jsdoc.app/tags-param.html

Anyways no big deal, was just curious.

Reply:
O good catch. We actually want to follow JSDoc syntax here. Have made the changes :+1: